packages/xrpl/src/models/index.ts

```diff
@@ -8,8 +8,9 @@
  */
 export * as LedgerEntry from './ledger'
 export {
-  setTransactionFlagsToNumber,
   parseAccountRootFlags,
+  setTransactionFlagsToNumber,
```

Comment:
Is it possible to remove this import of `setTransactionFlagsToNumber` ? That would ensure that `packages/xrpl/src/models/*` will always use the newer pure function.

Reply:
This was intentionally left in since removing it would create a breaking change. To prevent this, I've added deprecated annotations that cause linting errors and a console warning to the method, and maybe it would be good to remove two releases from now (one so the community can see the deprecation warning and react, another to actually remove)